core/src/main/java/com/linecorp/armeria/client/ClientFactoryBuilder.java

```diff
@@ -231,6 +234,39 @@ public ClientFactoryBuilder http2MaxFrameSize(int http2MaxFrameSize) {
     }
 
     /**
+     * Sets the default maximum length of the initial line.
+     */
+    public ClientFactoryBuilder defaultMaxInitialLineLength(int defaultMaxInitialLineLength) {
+        checkArgument(defaultMaxInitialLineLength < 0,
+                "defaultMaxHeaderSize: %s (expected: >= 0)",
+                defaultMaxInitialLineLength);
+        this.defaultMaxInitialLineLength = defaultMaxInitialLineLength;
+        return this;
+    }
+
+    /**
+     * Sets the default maximum length of all headers.
+     */
+    public ClientFactoryBuilder defaultMaxHeaderSize(int defaultMaxHeaderSize) {
+        checkArgument(defaultMaxHeaderSize < 0,
+                "defaultMaxHeaderSize: %s (expected: >= 0)",
+                defaultMaxHeaderSize);
+        this.defaultMaxHeaderSize = defaultMaxHeaderSize;
+        return this;
+    }
+
+    /**
+     * Sets the default maximum length of the content or each chunk.
```

Comment:
- the default maximum -> the maximum
- ... of each chunk in an HTTP/1 response content. The content or a chunk longer than this value will be split into smaller chunks so that their lengths never exceeds it.